src/common/init.cpp

```diff
@@ -31,15 +31,16 @@ namespace peloton {
 ThreadPool thread_pool;
 
 void PelotonInit::Initialize() {
-  QUERY_THREAD_COUNT = std::thread::hardware_concurrency();
+  CONNECTION_THREAD_COUNT = std::thread::hardware_concurrency();
   LOGGING_THREAD_COUNT = 1;
   GC_THREAD_COUNT = 1;
   EPOCH_THREAD_COUNT = 1;
+  MAX_CONCURRENCY = 10;
```

Comment:
yes it is. the users should be responsible for setting the maximum number of connections (threads) in the system.